docs/src/main/docs/tutorials/grpc-server-client.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: docs
+title: gRPC server and client
+section: tutorials
+permalink: tutorials/grpc-server-client
+---
+
+# Tutorial: gRPC server and client
+
+This tutorial will show you how to implement a working gRPC server and client
+based on a Mu service defintion.
+
+This tutorial is aimed at developers who:
+
+* are new to Mu-Scala
+* have some understanding of [cats-effect]
+* have read the [Getting Started guide](../getting-started)
+* have followed either the [gRPC with Protobuf](grpc-with-protobuf) or [gRPC
+  with Protobuf](grpc-with-protobuf) tutorial
+
+Mu supports both Protobuf and Avro. For the purposes of this tutorial we will
+assume you are using Protobuf, but it's possible to follow the tutorial even if
+you are using Avro.
+
+## Service definition
+
+If you have followed one of the previous tutorials, you should already have a
+service definition that looks like this:
```

Comment:
Not a big deal, but maybe mentioning that this particular code is related to protobuf?